src/pages/SessionCreator.tsx

```diff
@@ -1,37 +1,77 @@
 import Stepper from "@/components/Stepper";
-import TestDetails from "@/components/Steps/TestDetails";
-import Timeline from "@/components/Steps/Timeline";
 import StudentDetails from "@/components/Steps/StudentDetails";
+import { TestDetails } from "@/components/Steps/TestDetails";
+import Timeline from "@/components/Steps/Timeline";
+import { RowType } from "@/types/types";
+import { useRouter } from "next/router";
+
+import { useState } from "react";
 
-import React, { useState } from "react";
 export enum Step {
   STUDENT_DETAILS = "StudentDetails",
   TEST_DETAILS = "TestDetails",
   TIMELINE = "Timeline",
 }
 
-//Renders the main page for the multiple sub-pages that are rendered after
+const stepArr: string[] = [
+  Step.STUDENT_DETAILS,
+  Step.TEST_DETAILS,
+  Step.TIMELINE,
+];
+
 export default function SessionCreator() {
+  const router = useRouter();
   const [activeStep, setActiveStep] = useState<string>(Step.STUDENT_DETAILS);
+  const [data, setData] = useState<RowType>({
+    student: {},
+    test: {},
+    timeline: {},
+  });
+  console.log(data!);
+
+  const createSession = () => {
```

Comment:
ideally for now this should add to a `database.json` sort-of file that mimicks a db